src/@types/parseable/api/stream.ts

```diff
@@ -41,6 +41,7 @@ export type StreamInfo = {
 	static_schema_flag: boolean;
 	time_partition_limit: string;
 	custom_partition: string;
+	stream_type: string;
```

Comment:
Get all the possible values from the backend and define the type here instead of just string